test/java/lang/ref/OOMEInReferenceHandler.java

```diff
@@ -52,9 +54,10 @@
      }
 
      public static void main(String[] args) throws Exception {
-         // preinitialize the InterruptedException class so that the reference handler
+         // preinitialize the InterruptedException and WispEngine class so that the reference handler
          // does not die due to OOME when loading the class if it is the first use
          InterruptedException ie = new InterruptedException("dummy");
+         boolean isTransparentAsync = WispEngine.transparentWispSwitch();
```

Comment:
what's the usage?

Reply:
As the comment above:
```
// preinitialize the InterruptedException class so that the reference handler
// does not die due to OOME when loading the class if it is the first use
```